### PR TITLE
Fix markdown lint issues in SETTING_BASH.md

### PR DESCRIPTION
**Pull request by ycaglar**
This commit fixes markdown lint issues in SETTING_BASH.md file under /troubleshooting as per issue #47

### CODE REVIEW

**Comment by kadler** [comment]
There are multiple changes in this PR. Please remove the README.md changes. You may want to open a new branch and pull in just the one commit and submit a new PR or just roll back one commit with `git reset --hard HEAD^`

File: troubleshooting/SETTING_BASH.md [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
+to be your default shell via one of the following techniques:
 
-** Technique #1: chsh**
+## Technique #1: chsh
 
 1. Use yum to install the `chsh` package (for instamce, `yum install chsh`)
-2. From a shell, use the `chsh` command to set your shell (for instance, `chsh -s /QOpenSys/pkgs/bin/bash`). You can set the shell for another user via the `-u` option (for instance, `chsh -s /QOpenSys/pkgs/bin/bash -u otherusr`)
 
-** Technique #2: sql**
+1. From a shell, use the `chsh` command to set your shell
```
Comment (kadler) [inline troubleshooting/SETTING_BASH.md:15]: This is part of the same numbered sequence and should not be reset to 1.

**Review by kadler — COMMENTED** [review]
Please remove the .DS_Store file from your commits.

File: troubleshooting/SETTING_BASH.md (hunk 1 above)
Comment (ycaglar) [reply troubleshooting/SETTING_BASH.md:15]: When list items are in increasing order, the lint tool complains `Ordered list item prefix` 
but if I change all list items to 1's then the file passes lint tests. Also, I realized that list items are actually in **increasing** order once the markdown file is rendered.

**Pull request merged**